### PR TITLE
convolutional.md done

### PR DESCRIPTION
**Pull request by shimazaki**
Hi Francois, I've done translation of convolutional.md. Sorry for late submission! - hideaki

### CODE REVIEW

File: sources/layers/convolutional.md [hunk 1]
```diff
@@ -29,48 +24,37 @@ model.add(Convolution1D(32, 3, border_mode='same'))
...
+- __nb_filter__: 使用する畳み込みカーネルの数（出力の次元）．
+- __filter_length__: それぞれのフィルターの（空間もしくは時間的な）長さ．
+- __init__: レイヤーの重みの初期化関数の名前（[initializations](../initializations.md)参照），
+	もしくは重み初期化に用いるTheano関数．このパラメータは`weights`引数を与えない場合にのみ有効です．
+- __activation__: 使用する活性化関数の名前（[activations](../activations.md)参照），
+	もしくは要素ごとのTheano関数．
+	もしなにも指定しなければ活性化は一切適用されません（つまり"線形"活性a(x) = x）．
+- __weights__: 初期重みとして設定されるnumpy配列のリスト．
+- __border_mode__: 'valid' あるいは 'same'．
+- __subsample_length__: 出力を部分サンプルするときの長さ．　<!--check-->
```
Comment (nzw0301) [inline sources/layers/convolutional.md]: I find a comment out in this line. 
You may remove it.

File: sources/layers/convolutional.md [hunk 2]
```diff
@@ -29,48 +24,37 @@ model.add(Convolution1D(32, 3, border_mode='same'))
...
+- __b_regularizer__: バイアス項に適用される[WeightRegularizer](../regularizers.md)のインスタンス．
+- __activity_regularizer__: ネットワーク出力に適用される[ActivityRegularizer](../regularizers.md)のインスタンス．
+- __W_constraint__: メインの重み行列に適用される[constraints](../constraints.md)モジュール（例えばmaxnorm, nonneg）のインスタンス．
+- __b_constraint__: バイアス項に適用される[constraints](../constraints.md)モジュールのインスタンス．
+- __bias__: バイアス項を含むかどうか（レイヤをアフィンにするか線形にするか）．
+- __input_dim__: 入力のチャネル/次元数．
+	このレイヤーがモデルの初めのレイヤーの場合，
+	この引数もしくはキーワード引数`input_shape`を指定する必要があります．
+- __input_length__: 入力系列が一定のときのその長さ．
+	この引数は上流の`Flatten`そして`Dense`レイヤーを繋ぐときに必要となります．　<!--check-->
```
Comment (nzw0301) [inline sources/layers/convolutional.md]: Same as above.

**Comment by shimazaki** [comment]
Thanks for checking. I removed them.

**Comment by fchollet** [comment]
Thanks a lot!

**Pull request merged**